README.md

```diff
@@ -8,11 +8,17 @@ The fastest and **easiest** way to automatically test and build any Unity projec
 
 ---
 
-#### Supported versions
+##### Supported versions
 
-It's generally considered good practice to use the same unity version for Unity Actions as you do to develop your project.
+It's generally considered good practice to use the same Unity version for Unity Actions as you do to develop your project.
 
-Check the [list](https://hub.docker.com/r/gableroux/unity3d/tags) of supported versions by Unity Actions. 
+Unity Actions are based on the 
+[unity3d](https://gitlab.com/gableroux/unity3d) 
+images from 
+[GabLeRoux](https://github.com/GabLeRoux). 
+Any version in this 
+[list](https://hub.docker.com/r/gableroux/unity3d/tags)
+can be used to test and build projects.
```

Comment:
@gableroux, I'm thinking of combining credits with supported versions at the top.

This would mean removing credits from the bottom of this file.

Could you please review this change and give me your feedback?

Reply:
I'm merging this PR, but this comment is still a sincere request for feedback.

Reply:
Looks good to me! 👍